plugins/orchestrator-common/scripts/openapi.sh

```diff
@@ -8,9 +8,9 @@ DEFINITION_FILE="./src/auto-generated/api/definition.ts"
 METADATA_FILE="./src/auto-generated/.METADATA.sha1"
 
 openapi_generate() {
-    npx openapi-typescript ${OPENAPI_SPEC_FILE} -o ${SCHEMA_FILE}
-    npx openapi-generator-cli generate -g asciidoc -i ./src/openapi/openapi.yaml -o ./src/auto-generated/docs/index.adoc
-    npx yaml2json -f ${OPENAPI_SPEC_FILE}
+    npx --yes openapi-typescript@6.7.5 ${OPENAPI_SPEC_FILE} -o ${SCHEMA_FILE}
+    npx --yes @openapitools/openapi-generator-cli@v2.13.1 generate -g asciidoc -i ./src/openapi/openapi.yaml -o ./src/auto-generated/docs/index.adoc
+    npx --yes yaml2json -f ${OPENAPI_SPEC_FILE}
```

Comment:
```suggestion
    npx --yes yaml2json@1.0.2 -f ${OPENAPI_SPEC_FILE}
```

Reply:
It doesn't work.
that yaml2json belongs to https://www.npmjs.com/package/js-yaml-cli
I tried to set the version but i failed: i think the package is too old and doesn't have support fro this. 
I tried to change package but i didn't find anything that works (they required older versions of nodejs for example)

Reply:
Try this instead:  
`npx --yes --package=js-yaml-cli@0.6.0 -- yaml2json -f ${OPENAPI_SPEC_FILE}`

Reply:
This works! Addressed
Thanks for the help